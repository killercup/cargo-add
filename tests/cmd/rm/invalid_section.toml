bin.name = "cargo-rm"
args = ["rm", "--build", "semver"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
