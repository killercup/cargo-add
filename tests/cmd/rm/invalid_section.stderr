    Removing semver from build-dependencies
Could not edit `Cargo.toml`.

ERROR: The table `build-dependencies` could not be found.
Command failed due to unhandled error: The table `build-dependencies` could not be found.

